js/utility/createButton.js

```diff
@@ -0,0 +1,8 @@
+export default function createButton(type, ICON, dataJob, elementId) {
```

Comment:
function should explicitly create a button as the name suggest. should it take a parameter `type`?